hikey_debian.mk

```diff
@@ -226,7 +226,7 @@ PERF_FLAGS := CROSS_COMPILE_HOST=$(CROSS_COMPILE_NS_USER) \
 	TA_DEV_KIT_DIR=$(OPTEE_OS_TA_DEV_KIT_DIR)
 
 aes-perf: optee-os optee-client
-	$(MAKE) -C -j$(NPROC) $(AESPERF_PATH) $(PERF_FLAGS)
+	$(MAKE) -j$(NPROC) -C $(AESPERF_PATH) $(PERF_FLAGS)
```

Comment:
+1, and in addition `-j$(NPROC)` should not have been there in the first place.
Indeed, GNU make handles the number of jobs automatically provided you always invoke sub-makefiles with `$(MAKE)` and not `make`.

So, please try `xtest --aes-perf --help`, it should work.